### ISSUE TITLE
Pipelines settings checkbox label has changed: "Allow anonymous access to badges"

### ISSUE THREAD

**Issue by softwarecraft** [MicrosoftDocs/vsts-docs#7074]
In section https://docs.microsoft.com/en-us/azure/devops/pipelines/create-first-pipeline#add-a-status-badge-to-your-repository, under "To configure the Allow anonymous access to badges setting", step 3. Select the **Allow anonymous access to badges** checkbox under General.

It is now called "Disable anonymous access to badges" and appears as a toggle slider not a checkbox.

Suggest you rephrase the doc text.

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 7336a04e-5b38-d02f-8bba-a55445b2f094
* Version Independent ID: 783c30b3-63b6-da7c-1336-b000bb8b4c66
* Content: [Create your first pipeline - Azure Pipelines](https://docs.microsoft.com/en-us/azure/devops/pipelines/create-first-pipeline?view=azure-devops&tabs=net%2Cbrowser%2Ctfs-2018-2#feedback)
* Content Source: [docs/pipelines/create-first-pipeline.md](https://github.com/MicrosoftDocs/vsts-docs/blob/master/docs/pipelines/create-first-pipeline.md)
* Product: **devops**
* Technology: **devops-cicd**
* GitHub Login: @steved0x
* Microsoft Alias: **sdanie**

**Comment by msebolt** [comment]
@softwarecraft Thank you for the feedback, please consider [becoming a contributor](https://docs.microsoft.com/contribute/) on future doc updates.

The documentation has been updated and is pending review/merge.

**Comment by atom88888** [comment]
Step 1 says to navigate to "project settings" and then step 2 says to open the "settings" tab under pipelines, but I see no slider for my organization's view?  Where is this mysterious setting?  Or is there some step I'm missing?

**Comment by brent-langdon** [comment]
> Step 1 says to navigate to "project settings" and then step 2 says to open the "settings" tab under pipelines, but I see no slider for my organization's view? Where is this mysterious setting? Or is there some step I'm missing? Is this a setting under the "Project"? the "repo" or the "pipeline"? I've looked in all of them and don't see it? Help!

Just to help anybody that stumbles across this...I am very new to Azure DevOps, and it took me a bit to figure out this one.  When you navigate to a Project in Azure DevOps, the Project Settings link is all the way down in the bottom left corner.  I was expecting to find it in the left menu or in a context menu somewhere.

From there the setting is Pipelines > Settings > Disable anonymous access to badges

The first time the setting was disabled with a hover message saying that it is being controlled at the Organization level. So I had to navigate to the Organization, click the Organization Settings in the lower left and toggle the setting.  I then navigated back to the Project and toggled the setting for the Project.  My badge is working now!

Given that this is a documentation page for people new to Azure DevOps and the non-standard UI design of sticking a settings icon way at the bottom of the page, it might be helpful to make the steps more clear.
